features/performance/assets/pre-boot-tuning.sh

```diff
@@ -128,5 +119,5 @@ else
     #rpm-ostree initramfs --enable --arg=-I --arg=/etc/systemd/system.conf
     #rpm-ostree initramfs --enable --arg=-I --arg=/etc/sysconfig/irqbalance
     
-    systemctl reboot
+    touch /var/reboot
```

Comment:
What is the reason to use that here ?

Reply:
the reboot systemd unit initiates the reboot only when this file is existing